### PR TITLE
protobuf-c 1.5.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
==> /home/linuxbrew/.linuxbrew/Cellar/knot/3.3.2/bin/kdig @94.140.14.140 www.knot-dns.cz +quic
  ;; WARNING: QUIC, failed to send
  ;; WARNING: can't send query to 94.[140](https://github.com/Homebrew/homebrew-core/actions/runs/6999118308/job/19038217457?pr=155605#step:4:141).14.140@853(QUIC)
  ;; WARNING: QUIC, failed to send
  ;; WARNING: can't send query to 94.140.14.140@853(QUIC)
  ;; WARNING: QUIC, failed to send
  ;; WARNING: can't send query to 94.140.14.140@853(QUIC)
  ;; ERROR: failed to query server 94.140.14.140@853(UDP)
```

File: Formula/p/protobuf-c.rb [hunk 1]
```diff
@@ -24,38 +23,11 @@ class ProtobufC < Formula
...
-  end
-
-  patch do
-    url "https://github.com/protobuf-c/protobuf-c/commit/66574f3fd85a205eb7c90b790477d5415364209e.patch?full_index=1"
-    sha256 "2d1d6edbd615dff4f0a9c4a974d325effefc44466e1855f8c0b88e5977962a9d"
-  end
-
   def install
-    # https://github.com/protocolbuffers/protobuf/issues/9947
-    ENV.append_to_cflags "-DNDEBUG"
```
Comment (carlocab) [inline Formula/p/protobuf-c.rb:55]: Might be safer to keep this, I think.

Comment (chenrui333) [reply Formula/p/protobuf-c.rb:55]: fair enough, I will revert the change :)

File: Formula/p/protobuf-c.rb [hunk 2]
```diff
@@ -24,38 +23,14 @@ class ProtobufC < Formula
...
-  end
-
   def install
     # https://github.com/protocolbuffers/protobuf/issues/9947
     ENV.append_to_cflags "-DNDEBUG"
 
-    # TODO: Uncomment `if build.head?` when the patches are no longer needed.
-    system "autoreconf", "--force", "--install", "--verbose" # if build.head?
+    system "autoreconf", "--force", "--install", "--verbose" if build.head?
     system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
```
Comment (SMillerDev) [inline Formula/p/protobuf-c.rb:34]: Let's also change this to `*std_configure_args`

Comment (chenrui333) [reply Formula/p/protobuf-c.rb]: fair point 👍

**Review by carlocab — APPROVED** [review]

**Review by carlocab — APPROVED** [review]

**Pull request merged**